src/components/SocialMediaLinks/SocialMediaLinks.mdx

```diff
@@ -0,0 +1,135 @@
+import {ArgTypes, Canvas, Description, Meta, Source} from "@storybook/blocks";
+import { GetTableData } from "./SocialMediaLinksUtils";
+import Table from "../Table/Table"
+import * as SocialMediaLinksStories from "./SocialMediaLinks.stories";
+import Link from "../Link/Link";
+
+<Meta of={SocialMediaLinksStories} />
+
+# SocialMediaLinks
+
+| Component Version | DS Version     |
+|-------------------|----------------|
+| Added             |  `Prerelease`  |
+| Latest            |  `Prerelease`  |
+
+## Table of Contents
+
+[//]: # ( @todo This TOC is pretty long. I don't see a DS component that allows you to create a collapsible list. Might be nice to have.)
```

Comment:
There's no DS component because we haven't considered its implications regarding accessibility or usability. This can be the entire list and it's not a problem if it's long.

Reply:
The `<details>` tag is considered accessible according to [this resource](https://kb.daisy.org/publishing/docs/html/details.html). But the `<summary>` tag language is suboptimal, I guess, since it does not describe the details contents w/o simply repeating the section head ("Table of Contents.") Further, I suppose it could be argued that the `<details>` tag is being misused since a TOC is not really a detail of another thing.

However, as a sighted developer, I think the fully expanded TOC is unhelpful as it pushes down the substantive content nearly below the fold.

Perhaps @bigfishdesign13 has an opinion.

Reply:
For now, let's use the same formatting and functionality on the TOC that we are using on the other story pages.  We can consider an updtae like this as a separate task.